src/style.css

```diff
@@ -1 +1,63 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
```

Comment:
You've done a good job of defining the `@font-face`, but you forgot to include a fallback font family. This is important for cases where the main font doesn't load correctly. You can add a generic font family like `sans-serif` as a fallback.